app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    _, current_file, new_file_path = command.split()
```

Comment:
What if user passes command wrong (for example uses wrong command or passes 2 or 4 args instead of 3 that you expect?)